snowflake/cmd/createBucket_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	sftesting "antrea.io/theia/snowflake/cmd/testing"
+	s3clienttesting "antrea.io/theia/snowflake/pkg/aws/client/s3/testing"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	s3types "github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+const bucketExistLog = "S3 bucket already exists"
+
+func TestCreateBucket(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	origLogger := logger
+	defer func() {
+		logger = origLogger
+	}()
+
+	var b bytes.Buffer
+	logger = sftesting.NewLogger(&b)
+
+	for _, tc := range []struct {
```

Comment:
in a case like this one where you are only going to gave 2 cases (new bucket / existing bucket) and where most of the test case code ends up being conditioned by `tc.bucketExist`, I would recommend _not_ using table-driven tests. You can just write:
```go
t.Run("Create existing bucket", func(t *testing.T) {
// ...
})
t.Run("Create new bucket", func(t *testing.T) {
// ...
})
```
It's more readable and not really more verbose